tests/test_measurements.py

```diff
@@ -73,6 +76,7 @@ def test_create_invalid_name(self):
         with self.assertRaises(KeyError):
             Measurement(metric, value, metric_set=self.metric_set)
 
+    @unittest.skip('FIXME DM8477 this is failing')
```

Comment:
That's fair. I think it raises a different (broader) exception in 1.2, so we could switch to that, or just leave it as a skip. We probably want to upgrade to 1.3 (or the point release that will be coming out soonish) anyway, for a variety of other reasons.